### PR TITLE
Release 8.35.0

### PR DESCRIPTION
**Pull request by gendeld**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -177,6 +177,7 @@
     "storybook-chromatic": "^3.5.3",
     "storybook-snapper": "^1.0.63",
     "sync-ts": "^1.0.67",
+    "thread-loader": "^2.1.2",
```
Comment (shlomitc) [inline package.json:180]: why is this added?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5774/
